tests/test_clinical_trials_dot_gov.py

```diff
@@ -3,7 +3,7 @@
 from pydantic import ValidationError
 import pytest
 
-from trial_synth.clinical_trials_dot_gov import (
+from src.trialsynth.clinical_trials_dot_gov import (
```

Comment:
src here doesn't make sense, the import should use the package name i.e., `trialsynth`